scss/functions/_get-color.scss

```diff
@@ -2,9 +2,23 @@
 @use 'sass:map';
 
 @function get-color($type, $level) {
-  // add the logic here
-  // use if else to check if the type and level are valid
-  // if they are valid, return the color
-  // if they are not valid warn 'Invalid color type or level' and return null
-  // null is a special value in Sass that means nothing, and it is used to represent the absence of a value, you need to return null if the type or level is not valid
+  @if $type == null or $level == null {
+    @return 'Error: Type or Level is missing';
+  }
```

Comment:
bagian ini lebih baik dihapus karena semua logic sudah ditangani oleh if else di bawah

Reply:
Semua telah saya perbaiki hanya tinggal bagian ini saja

Concern saya tulis fungsi itu adalah in case ada kurang tulis sesuatu maka keluar warning
misal hanya tulis gray tanpa levelnya

Saya hapus aja? 
ato saya ganti warn dan return null?

terima kasih
![image](https://github.com/user-attachments/assets/7cd74660-4264-4e16-a05f-9a1ffd716630)